### DISCUSSION TITLE
broadcast.owncast.online

### DISCUSSION THREAD

**Discussion by ghost** [owncast/owncast#3856] [General Owncast discussion]
Hi! Your address [broadcast.owncast.online](http://broadcast.owncast.online/) is not working! Please resolve.

**Comment by gabek** [comment]
Hi there. That server was actually decommissioned a while ago, as we needed to downsize the number of resources we were running. Could you share where you found the link so we can remove it?

**Reply by ghost** [reply]
I found here with words "Stream your content to" https://owncast.online/demo/

**Comment by gabek** [comment]
Thanks!

**Comment by germainelee** [comment]
Update: this has been resolved in https://github.com/owncast/owncast/issues/3860
